packages/frontend/src/Build.spec.ts

```diff
@@ -669,7 +663,7 @@ test('have amd64 and arm64 NOT disabled if inspectManifest contains both archite
   render(Build, { imageName: 'testmanifest1', imageTag: 'latest' });
 
   await vi.waitFor(() => {
-    expect(spyOnInspectManifest).toHaveBeenCalled();
+    expect(vi.mocked(bootcClient.inspectManifest)).toHaveBeenCalled();
```

Comment:
```suggestion
    expect(bootcClient.inspectManifest).toHaveBeenCalled();
```

Can be simplified: vite is very nice for helping us and directly consider a mock when we pass it to expect, it would throw an error if the object is not an mock

Reply:
👍🏼 Fixed.